app/application.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../lib/converter'
+
+class Application
+    def call(env)
+        response = Rack::Response.new
+        request  = Rack::Request.new(env)
+
+        if request.path.match(/convert/)
+            numeral = request.params["roman"]
+
+            converted_numeral = Converter.new.convert_roman_to_arabic(numeral)
+            result            = JSON.parse(converted_numeral)
+
+            response.status   = 200
+            response.write "\n"
+            response.write result["value"]
```

Comment:
Let's have the endpoint return a valid JSON string in the response body. Something like:

```
curl localhost:8000/convert&roman=IV
# {
#   "value": 4
# }
```